distribution/test/thirdparty/rest-assured/pom.xml

```diff
@@ -27,6 +27,7 @@
             <groupId>com.jayway.restassured</groupId>
             <artifactId>rest-assured</artifactId>
             <version>${restassured.version}</version>
+            <scope>test</scope>
```

Comment:
The bamboo  build failed with the following error: 
19-Jan-2016 15:40:22	[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:testCompile (default-testCompile) on project test-itests-common: Compilation failure: Compilation failure:
19-Jan-2016 15:40:22	[ERROR] /home/bamboo/bamboo-agent-home/xml-data/build-dir/DDFPR-ROOT-JOB1/ddf/distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java:[19,37] package com.jayway.restassured does not exist
19-Jan-2016 15:40:22	[ERROR] /home/bamboo/bamboo-agent-home/xml-data/build-dir/DDFPR-ROOT-JOB1/ddf/distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java:[19,1] static import only from classes and interfaces
19-Jan-2016 15:40:22	[ERROR] /home/bamboo/bamboo-agent-home/xml-data/build-dir/DDFPR-ROOT-JOB1/ddf/distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java:[56,39] package com.jayway.restassured.response does not exist

Does the failure have something to do with changing the scope to test?